applications/serial_lte_modem/src/main.c

```diff
@@ -357,6 +358,75 @@ void handle_mcuboot_swap_ret(void)
 	slm_setting_fota_save();
 }
 
+int nidd_auto_connect(void)
+{
+	int err = 0;
+
+#if defined(CONFIG_LWM2M_CARRIER_SERVER_BINDING_N)
+	int cfun;
+
+	if (!fota_reconnect) {
+		return 0;
+	}
+	err = nrf_modem_at_scanf("AT+CFUN?", "+CFUN: %d", &cfun);
+	if (err != 1 || (cfun == 1 || cfun == 5)) {
```

Comment:
Do we support cfun=5?

Reply:
5 is "roaming" status in the AT READ result.

Reply:
@junqingzou You are probably mixing [AT+CFUN](https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/mob_termination_ctrl_status/cfun_read.html) and [AT+CEREG](https://infocenter.nordicsemi.com/topic/ref_at_commands/REF/at_commands/nw_service/cereg_read.html).

Reply:
What a shame! 🤦‍♂️